deploy/test/test_cases/TEST_ID_25_helm_default_retry_successful.sh

```diff
@@ -36,18 +36,18 @@ $cli_with_timeout "logs $pod_name | grep 'CSPFK010E Failed to authenticate'"
 # Start the timer for retry interval
 start=$SECONDS
 
-echo "Expecting Secrets Provider retry configurations to take defaults RETRY_INTERVAL_SEC 30 and RETRY_COUNT_LIMIT 3"
+echo "Expecting Secrets Provider retry configurations to take defaults RETRY_INTERVAL_SEC $DEFAULT_RETRY_INTERVAL_SEC and RETRY_COUNT_LIMIT $DEFAULT_RETRY_COUNT_LIMIT"
 $cli_with_timeout "logs $pod_name | grep 'CSPFK010I Updating Kubernetes Secrets: 1 retries out of $DEFAULT_RETRY_COUNT_LIMIT'"
 
 duration=$(( SECONDS - start ))
 # Since we are testing retry in scripts we must determine an acceptable range that retry should have taken place
 # If the duration falls within that range, then we can determine the retry mechanism works as expected
-retryIntervalMin=`echo "scale=2; $DEFAULT_RETRY_INTERVAL_SEC/100*80" | bc | cut -d "." -f 1 | cut -d "," -f 1`
-retryIntervalMax=`echo "scale=2; $DEFAULT_RETRY_INTERVAL_SEC/100*120" | bc | cut -d "." -f 1 | cut -d "," -f 1`
-if (( $duration >= $retryIntervalMin && $duration <= $retryIntervalMax )); then
-  echo 0
+retryIntervalMin=`echo "scale=3; 1/100*80" | bc -l | awk '{print ($0-int($0)<0.499)?int($0):int($0)+1}'`
+retryIntervalMax=`echo "scale=3; 1/100*120" | bc -l | awk '{print ($0-int($0)<0.499)?int($0):int($0)+1}'`
```

Comment:
why not using $RETRY_INTERVAL_SEC/100...

Reply:
great catch